amplify/backend/function/wandb/wandb-cloudformation-template.json

```diff
@@ -95,7 +95,7 @@
         "Layers": [
           "arn:aws:lambda:us-east-1:770693421928:layer:Klayers-p38-numpy:13"
         ],
-        "Timeout": 25
+        "Timeout": 120
```

Comment:
https://docs.amplify.aws/javascript/build-a-backend/functions/configure-options/

Time out was needed to be updated 

<img width="502" alt="image" src="https://github.com/surcyf123/pretraining-frontend/assets/25290212/35ea3dea-2a88-46b4-8b1f-e28800f53b8e">